guides/doc-Upgrading_and_Updating/topics/proc_upgrading-a-connected-project-server.adoc

```diff
@@ -12,6 +12,19 @@ include::../common/modules/snip_using_installer_noop.adoc[]
 * On a virtual machine, take a snapshot.
 * On a physical machine, create a backup.
 +
+Note that when creating a snapshot or conventional backup, you must stop all services as follows:
```

Comment:
Would it make sense to turn all this into regular procedure steps?

1. Stop all services.
2. Take a snapshot/create a backup.
3. Start the services.

Reply:
ack @asteflova yes! I was wondering what would be a better way to arrange this information. Thanks :)